clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/BaseGVFSOperations.java

```diff
@@ -369,6 +376,15 @@ protected long defaultBlockSize() {
     return defaultBlockSize;
   }
 
+  /**
+   * Whether to enable credential vending.
+   *
+   * @return true if credential vending is enabled, false otherwise.
+   */
+  protected boolean enableCredentialVending() {
```

Comment:
why this method is `protected`?

Reply:
when the developers implement their own GVFS ops, they can call the method to know whether the credential vending is enabled

Reply:
ok